benchmarks/communication/all_gather.py

```diff
@@ -1,7 +1,9 @@
 '''Copyright The Microsoft DeepSpeed Team'''
 
-import torch
-import sys, os, time
+from benchmarks.communication.utils import *
```

Comment:
These imports were moved down to https://github.com/microsoft/DeepSpeed/pull/2995/files#diff-57ff0beda6888da8839b4b96071460313d9ad4d5b6ce148337374409602b9403R11. 

`from deepspeed.comm import TorchBackend` should be moved to L14, and the rest in this block can be deleted.

Reply:
> @mayank31398, I think the formatting issues can be fixe by upgrading pre-commit and clang-format

i am not seeing any issues with the formatting in the CI.
are you suggesting to update the pre-commit nonetheless @tjruwase ?

Reply:
It's still off. The beginning of this file should look like:

```
'''Copyright The Microsoft DeepSpeed Team'''

import torch
import sys, os, time

COMMS_BENCH_DIR = os.path.join(os.path.dirname(__file__), "../")
sys.path.append(COMMS_BENCH_DIR)

from communication.utils import *
from communication.constants import *
from deepspeed.accelerator import get_accelerator
from deepspeed.comm import TorchBackend
```

Reply:
done

Reply:
Great! I'm happy with this. Thanks for sticking through it @mayank31398 :)

Reply:
Awesome, lets get this in. Don't wanna rebase against master again :)